src/theme/components/header.ts

```diff
@@ -0,0 +1,122 @@
+const container = {
+  fontFamily: "'system ui', 'Segoe UI', Tahoma, 'Helvetica', 'arial'",
+};
+
+const notification = {
+  bg: "#fee249",
+  width: "100%",
+  "div > .css-1jloxoz": {
+    fontSize: "var(--nypl-fontSizes-text-tag)",
+    letterSpacing: ".01em",
+    lineHeight: "1.25em",
+    margin: "0.5em 1.2em 0.5em 1.2em",
+  },
+};
+
+const logo = {
+  marginRight: "auto",
+};
+
+const upperLinks = {
+  alignItems: "center",
+  marginTop: "var(--nypl-space-s)",
+  marginLeft: "auto",
+  whiteSpace: "nowrap",
+  li: {
+    bgColor: "#ffffff",
+    fontSize: "var(--nypl-fontSizes-text-mini)",
+    fontWeight: 600,
+    letterSpacing: ".05em",
+    marginLeft: "var(--nypl-space-xs)",
+    textTransform: "uppercase",
+  },
+  "li > a": {
+    color: "#000000",
+    textDecoration: "none",
+  },
+  "li > button": {
+    alignItems: "center",
+    fontSize: "var(--nypl-fontSizes-text-mini)",
+    fontWeight: 600,
+    letterSpacing: ".05em",
+    textTransform: "uppercase",
+  },
+  "li > .withIcon": {
+    color: "#000000",
+    textDecoration: "none",
+    svg: {
+      position: "relative",
+      height: "5px",
+      top: -0.5,
+      right: 0.5,
+    },
+  },
+  "li > #donateButton": {
+    bg: "#ed1c24",
```

Comment:
Make the donate button `buttonType={ButtonTypes.Callout}`, but keep this bg color here. If you need to, you might need an `!important` so this sticks.